p2p/simulations/mocker.go

```diff
@@ -67,6 +67,10 @@ func startStop(net *Network, quit chan struct{}, nodeCount int) {
 	}
 	tick := time.NewTicker(10 * time.Second)
 	defer tick.Stop()
+
+	timer := time.NewTimer(3 * time.Second)
+	defer timer.Stop()
```

Comment:
```suggestion
var (
	tick = time.NewTicker(10 * time.Second)
	timeout = time.NewTimer(3 * time.Second)
)
defer tick.Stop()
defer timeout.Stop()
```
maybe something like this?